demo/docker-compose.yaml

```diff
@@ -43,13 +43,9 @@ grafana:
   links:
   - clickhouse
 grafana-key:
-  image: jfloff/alpine-python:3.6-onbuild
+  build: grafana-key
```

Comment:
do you build this image during your script execution? maybe it is better to just use `python-alpine` image and `CMD` something like `sh -c 'pip install requests && python get-key.py'`

Reply:
Fixed